UPGRADE-2.0.md

```diff
@@ -1114,374 +373,1280 @@ In Sylius 2.0, we have changed the visibility of services to `private` by defaul
-
-## Payment method gateways
-
-* Stripe gateway has been removed. This implementation has been deprecated and not SCA Ready.
-* PayPal Express Checkout gateway has been removed. Use now [PayPal Commerce Platform](https://github.com/Sylius/PayPalPlugin) integration.
-
-## Theming
-* Dependency on `sylius/theme-bundle` is moved from CoreBundle to ShopBundle and it will no longer be installed 
-  if you're running your shop in headless mode.
-* Channel's `themeName` form field existence is made optional and depends on `ShopBundle` presence.
-* The `Sylius\Bundle\CoreBundle\Theme\ChannelBasedThemeContext` has been moved to the `Sylius\Bundle\ShopBundle\Theme\ChannelBasedThemeContext`.
-
-## Routing
-
-* Routing for the `sylius_shop_payum` has been moved from the `SyliusShopBundle` to the `SyliusPayumBundle`:
-
-```yaml
-sylius_shop_payum:
--   resource: "@SyliusShopBundle/Resources/config/routing/payum.yml"
-+   resource: "@SyliusPayumBundle/Resources/config/routing/integrations/sylius_shop.yaml"
-```
-
-## Testing Suite
+#### KNP Menu
 
-* The `sylius.behat.api_security` has been replaced by `sylius.behat.api_admin_security` and `sylius.behat.api_shop_security` services.
+* Aliases for the `knp_menu.menu_builder` tags introduced in Sylius 1.14 are now the only valid menu builder tags in
+  Sylius 2.0:
 
-## Templates changes
+| Old Alias             | New Alias             |
+|-----------------------|-----------------------|
+| **AdminBundle**       |                       |
+| `sylius.admin.main`   | `sylius_admin.main`   |
+| **ShopBundle**        |                       |
+| `sylius.shop.account` | `sylius_shop.account` |
 
-### Transition from SemanticUI to Bootstrap
+#### Transition from SemanticUI to Bootstrap
 
 - All `.ui` prefixed classes have been replaced with Bootstrap classes.
 - JavaScript components relying on Semantic UI have been rewritten to utilize Bootstrap's JavaScript plugins.
 - Customized CSS has been replaced by Bootstrap's utility classes.
 
-### Abandoning jQuery
+#### Abandoning jQuery
 
 Most of the existing JavaScript has been replaced by SymfonyUX with Stimulus, which includes live components.
 This change led to the removal of jQuery and a significant reduction of custom JavaScript in the project. 
 Check out the documentation for more information [here](https://ux.symfony.com/).
 
-### Abandoning partial routes
+#### Abandoning partial routes
 
 All partial routes rendered in templates have been removed and replaced by rendering Twig components.
 
-### Sylius Twig Hooks
+#### Sylius Twig Hooks
 
 Twig Hooks are a robust and powerful alternative to the Sonata Block Events and the old Sylius Template Events systems.
 
-#### Abandoning Sonata Blocks
+##### Abandoning Sonata Blocks
 
 Sonata Blocks have been fully replaced with Twig Hooks.
 
-#### Evolving Sylius Template Events to Twig Hooks
+##### Evolving Sylius Template Events to Twig Hooks
```

Comment:
Maybe add a note regarding the templates directory structure as well?
Since we have a pretty much 1:1 regarding hook name:template location, would be nice to flesh it out.
Probably will also help with the upgrade when people know more or less where the parts they need to change are located/how the appropriate hook can be named.